db/schema.sql

```diff
@@ -44,6 +41,17 @@ CREATE TABLE task_hpke_keys(
     CONSTRAINT fk_task_id FOREIGN KEY(task_id) REFERENCES tasks(id)
 );
 
+-- The VDAF verification parameters used by a given task.
+CREATE TABLE task_vdaf_verify_params(
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,  -- artificial ID, internal-only
+    task_id BIGINT NOT NULL,           -- task ID the verification parameter is associated with
+    ord BIGINT NOT NULL,               -- a value used to specify the ordering of verification parameters
```

Comment:
I'm a little nervous about implementing that the VDAF parameters are ordered before we specify/know how multiple verify parameters are used. I think we need to do _something_ specification-wise, and I strongly suspect whatever we do will end up communicating a "verify_parameter ID" of some kind (since the helper would otherwise have to guess, or try them all which is expensive), which may well not match up to the `ord` parameter we implement here.

WDYT about dropping the `ord` parameter, and making the `UNIQUE` constraint cover only `task_id`? This would enforce one-verify-param-per-task at a DB level, which I think is a good idea since that's what we want for now.

Reply:
Hmm. This does have the downside that we could no longer include `ord` in the crypter "row" parameter -- we'd need to do a data migration if/when we did introduce whatever verify parameter ID ends up being specified.

Still--if we wanted to switch from `ord` to a specified ID, we might need a data migration anyway, so maybe this point is moot?

Reply:
Guess-and-check on verification parameters wouldn't work, because we could only learn if we chose the "right" one when the leader combines its prepare message with the helper's prepare message, and applies the "decide" function to the result. The aggregators will either need to communicate some identifier with each aggregate request or select parameters a priori, for example by assigning a range of timestamps to each verification parameter.

Ideally, if there is any verification parameter identifier exposed in the protocol, we should use that in the AAD when encrypting the verification parameter. In that case, there's no way around a migration. I'll try out making the `UNIQUE` change.